# Data Analysis - Video Game Sales - Charles Glass - 12/16/2019

In [2]:
import pandas as pd

In [3]:
import numpy as np

In [5]:
df = pd.read_csv('vgsales.csv')

In [ ]:
df

In [ ]:
df[['Publisher']].mode()

the most common video game publisher is electronic arts

In [ ]:
df[['Platform']].mode()

the most common platform is the DS

In [ ]:
df[['Genre']].mode()

The most common Genre is Action

In [107]:
df.sort_values('Global_Sales', ascending= False)['Name'].head(21)

0                                       Wii Sports
1                                Super Mario Bros.
2                                   Mario Kart Wii
3                                Wii Sports Resort
4                         Pokemon Red/Pokemon Blue
5                                           Tetris
6                            New Super Mario Bros.
7                                         Wii Play
8                        New Super Mario Bros. Wii
9                                        Duck Hunt
10                                      Nintendogs
11                                   Mario Kart DS
12                     Pokemon Gold/Pokemon Silver
13                                         Wii Fit
14                                    Wii Fit Plus
15                              Kinect Adventures!
16                              Grand Theft Auto V
17                   Grand Theft Auto: San Andreas
18                               Super Mario World
19    Brain Age: Train Your Bra

The top twenty games are ^ (dont want to type out 20 games)

In [ ]:
df[['NA_Sales']].median()

the median North American sales is .08

In [ ]:
around_median = df['NA_Sales'].between(0.8, 0.8,)

In [ ]:
df[around_median]

Values around the median North American Sales ^

In [ ]:
gsmean = df[['NA_Sales']].mean()
other = df.loc[:, 'NA_Sales'].std()
other-gsmean

the top selling NA game is .55 above the mean

In [6]:
nintendo_sales = df[df['Publisher'].between('Nintendo', 'Nintendo')].mean()
other_sales = df[df['Publisher'].str.contains('Nintendo') == False].mean()
nintendo_sales.Global_Sales-other_sales.Global_Sales

2.09182017263173

Nintendo's mean sales is 2.09 above all other platforms

1st question: What are the top 5 worst selling Games?

In [ ]:
df.sort_values('Global_Sales', ascending= False)['Name'].tail(5)

question 2: what genre is the least popular?

In [106]:
df.sort_values('Genre', ascending= False)['Genre'].head(1)

11820    Strategy
Name: Genre, dtype: object

question 3: what are the top 5 grossing games in Japan

In [ ]:
df.sort_values('JP_Sales',  ascending= False)['Name'].head(5)